### PR TITLE
Created binarysearch.cpp

### PR DESCRIPTION
**Pull request by visakh2**

### CODE REVIEW

File: Projects/binarysearch.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#include <iostream>
```
Comment (ritiksoni00) [inline Projects/binarysearch.cpp:1]: Add comments!

Comment (ritiksoni00) [reply Projects/binarysearch.cpp:1]: You are supposed to make changes in Files @visakh2. i will close It After 8.00 pm! 
Please Do it!

File: Projects/binarysearch.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	int count, i, arr[30], num, first, last, middle;
+	cout<<"how many elements would you like to enter?:"; 
+        cin>>count;
+
+	for (i=0; i<count; i++)
+	{
+		cout<<"Enter number "<<(i+1)<<": "; 
+                cin>>arr[i];
+	}
+	cout<<"Enter the number that you want to search:";
```
Comment (thevirtualbuddy) [inline Projects/binarysearch.cpp:15]: Sort the numbers after taking input from the user

**Review by thevirtualbuddy — CHANGES_REQUESTED** [review]
Binary Search works only for sorted elements, so either
1. ask the user to input the elements in sorted order (not recommended)
or
2. sort the elements after the user inputs the data (recommended). 

One example where your code fails is shown below:

how many elements would you like to enter?:5
Enter number 1: 1
Enter number 2: 6
Enter number 3: 2
Enter number 4: 99
Enter number 5: 4
Enter the number that you want to search:4
4 not found in the array

**Pull request closed without merging**